external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -105,70 +102,86 @@ fn main() {
+                },
+                resolve_provider: None,
+            },
+        ))),
         ..Default::default()
     })
     .expect("could not serialize server capabilities");
 
     let (diag_sender, diag_receiver) = bounded::<Result<BTreeMap<PathBuf, Vec<Diagnostic>>>>(0);
-    let mut symbolicator_runner = symbols::SymbolicatorRunner::idle();
-    if symbols::DEFS_AND_REFS_SUPPORT {
-        let initialize_params: lsp_types::InitializeParams =
-            serde_json::from_value(client_response)
-                .expect("could not deserialize client capabilities");
+    let initialize_params: lsp_types::InitializeParams =
+        serde_json::from_value(client_response).expect("could not deserialize client capabilities");
 
-        // determine if linting is on or off based on what the editor requested
-        let lint = {
-            let lint_level = initialize_params
-                .initialization_options
-                .as_ref()
-                .and_then(|init_options| init_options.get("lintLevel"))
-                .and_then(serde_json::Value::as_str)
-                .unwrap_or(LINT_DEFAULT);
-            if lint_level == LINT_ALL {
-                LintLevel::All
-            } else if lint_level == LINT_NONE {
-                LintLevel::None
-            } else {
-                LintLevel::Default
-            }
-        };
-        eprintln!("linting level {:?}", lint);
+    // determine if linting is on or off based on what the editor requested
+    let lint = {
+        let lint_level = initialize_params
+            .initialization_options
+            .as_ref()
+            .and_then(|init_options| init_options.get("lintLevel"))
+            .and_then(serde_json::Value::as_str)
+            .unwrap_or(LINT_DEFAULT);
+        if lint_level == LINT_ALL {
+            LintLevel::All
+        } else if lint_level == LINT_NONE {
+            LintLevel::None
+        } else {
+            LintLevel::Default
+        }
+    };
+    eprintln!("linting level {:?}", lint);
 
-        symbolicator_runner = symbols::SymbolicatorRunner::new(
-            ide_files_root.clone(),
-            symbols.clone(),
-            pkg_deps.clone(),
-            diag_sender,
-            lint,
-        );
+    let symbolicator_runner = symbols::SymbolicatorRunner::new(
+        ide_files_root.clone(),
+        symbols.clone(),
+        pkg_deps.clone(),
+        diag_sender,
+        lint,
+    );
 
-        // If initialization information from the client contains a path to the directory being
-        // opened, try to initialize symbols before sending response to the client. Do not bother
-        // with diagnostics as they will be recomputed whenever the first source file is opened. The
-        // main reason for this is to enable unit tests that rely on the symbolication information
-        // to be available right after the client is initialized.
-        if let Some(uri) = initialize_params.root_uri {
```

Comment:
The `root_uri` is the deprecated field but I am not immediately sure how to replace it